docs-source/content/release-notes.md

```diff
@@ -39,6 +39,37 @@ draft: false
 
 #### Operator 3.2.0
 
+##### Features
+
+* The operator's container image is based on Oracle Linux 8.
+* WebLogic Server container images based on Oracle Linux 8 are supported.  
+* [Online updates]({{<relref "/userguide/managing-domains/model-in-image/runtime-updates/_index.md#online-updates">}}) of dynamic configuration changes for Model in Image.
+* Automatic injection of the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) as a sidecar container.  
+* [Events]({{< relref "/userguide/managing-domains/domain-events.md" >}}) are generated at important moments in the lifecycle of the operator or a domain.
+* [PodDisruptionBudgets]({{<relref "/userguide/introduction/architecture/_index.md#draining-a-node-and-poddisruptionbudget">}}) are generated for clusters improving the ability to maintain cluster availability during planned node shutdowns and Kubernetes upgrade.
+* Additional scripts to assist with common tasks, such as the scaleCluster.sh script.
+* Support for TCP and UDP on the same channel when the SIP protocol is used.
+
+##### Fix for Bug or Regression
```

Comment:
Fix for Bug or Regression -> (We prefer most titles to be plural, therefore, Fixes for Bugs or Regressions. Other suggestion is Fixed Bugs or Regressions.)